plugins/woocommerce/includes/tracks/class-wc-site-tracking.php

```diff
@@ -64,12 +64,17 @@ public static function enqueue_scripts() {
 	 * Adds the tracking function to the admin footer.
 	 */
 	public static function add_tracking_function() {
+		$user           = wp_get_current_user();
+		$server_details = WC_Tracks::get_server_details();
+		$blog_details   = WC_Tracks::get_blog_details( $user->ID );
+
+		$client_tracking_properties = array_merge( $server_details, $blog_details );
```

Comment:
Instead of relying on the filter previously, we are now getting these properties straight from `WC_Tracks`.